sandbox/payment_method_test.go

```diff
@@ -0,0 +1,131 @@
+package sandbox
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/gtforge/go-zooz"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"testing"
+)
+
+func TestPaymentMethod(t *testing.T) {
+	t.Parallel()
+	client := GetClient(t)
+
+	t.Run("new & get & get-list", func(t *testing.T) {
+		t.Parallel()
+
+		customer := PrepareCustomer(t, client)
```

Comment:
Should we cleanup the created resources after test is finished (ignoring any cleanup errors)? 
You can use t.Cleanup inside `PrepareXXX` methods.

Reply:
Great question. Then we should clean up not only after `PrepareXXX` but after all tests.
Don't you mind if I do it in the next PR?

Reply:
ok

Reply:
I think we should cleanup nothing.
We can't delete:
* tokens
* payments
* transactions

We can delete only:
* customers
* their payment methods

But why, it we can't clean up all entities?

Reply:
makes sense